### PR TITLE
[libjpeg-turbo/mozjpeg] Add conflict messages

### LINKED ISSUE #13323: Error on installing tiff with mozjpeg
mozjpeg(libjpeg-turbo's fork) installed, tiff installing is fail.

```
Computing installation plan...
The following packages will be built and installed:
  * libjpeg-turbo[core]:x64-windows-static
    tiff[core]:x64-windows-static
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows-static...
Starting package 1/2: libjpeg-turbo:x64-windows-static
Building package libjpeg-turbo[core]:x64-windows-static...
Using cached binary package: C:\Users\daram\AppData\Local\vcpkg/archives\6f\6f50c25f613180f65162c2089642da02fdd392f1.zip
Building package libjpeg-turbo[core]:x64-windows-static... done
Installing package libjpeg-turbo[core]:x64-windows-static...
The following files are already installed in D:/Projects/vcpkg/installed/x64-windows-static and are in conflict with libjpeg-turbo:x64-windows-static

Installed by mozjpeg:x64-windows-static
    debug/lib/pkgconfig/libjpeg.pc
    debug/lib/pkgconfig/libturbojpeg.pc
    include/jconfig.h
    include/jerror.h
    include/jmorecfg.h
    include/jpeglib.h
    include/turbojpeg.h
    lib/jpeg.lib
    lib/pkgconfig/libjpeg.pc
    lib/pkgconfig/libturbojpeg.pc
    lib/turbojpeg.lib
```

### PR DESCRIPTION
**Pull request by NancyLi1013**
**Describe the pull request**

- What does your PR fix? Fixes #13323

`libjpeg-turbo` and `mozjpeg` cannot be installed at the same time, since `mozjpeg `is based on` libjpeg-turbo`.

The references are as follows:
https://github.com/mozilla/mozjpeg/blob/master/README.md
https://github.com/mozilla/mozjpeg/blob/master/README-mozilla.txt

Note: No need to test features.

### CODE REVIEW

**Comment by NancyLi1013** [comment]
@BillyONeal  
Could you please help take a look what's wrong on CI pipelines in this PR?
Thanks.

**Comment by BillyONeal** [comment]
@NancyLi1013 It fails vcpkg's end to end tests:

```
Failed at [../src/vcpkg-test/manifests.cpp(635)] with message:
value was error
```

if I had to guess because you said `Port-version` rather than `Port-Version` in the `CONTROL` here.

File: ports/mozjpeg/CONTROL [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Source: mozjpeg
-Version: 2020-06-02-1
+Version: 2020-06-02
+Port-version: 2
```
Comment (NancyLi1013) [inline ports/mozjpeg/CONTROL:3]:
```suggestion
Port-Version: 2
```

**Comment by NancyLi1013** [comment]
Thanks for your help to pointing this out @BillyONeal. 

Sorry, it's my mistake. Hope all checks pass this time.

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
Maybe we should add a `vcpkg_conflict_port` function instead.

**Pull request merged**